# don't create /usr/etc dir when --sysconfdir=/etc passed to configure

--- yaws-1.96/scripts/Install
+++ yaws-1.96/scripts/Install.new
@@ -18,7 +18,7 @@
 v=${vardir}
 
 install -d ${destdir}${prefix}/bin
-install -d ${destdir}${prefix}/etc
+install -d ${destdir}${etcdir}
 install -d ${destdir}${vardir}/log/yaws
 
 cat yaws.template | \
